sdk/textanalytics/azure-ai-textanalytics/src/samples/README.md

```diff
@@ -11,7 +11,7 @@ urlFragment: textanalytics-java-samples
 
 # Azure Text Analytics client library samples for Java
```

Comment:
do you happen to know if we need to add a new value under `products` to link the language service?

Reply:
I have no idea. @deyaaeldeen  @kristapratico do you have ideas?

Reply:
Good question! I don't know tbh but this is what is being used in JS: https://github.com/Azure/azure-sdk-for-js/blob/4c5b90b518a554fa4a7635541d20820634647bd7/sdk/textanalytics/ai-text-analytics/package.json#L48-L52. Let me start a quick thread.